CDCgov/prime-reportstream
# Filter submitters with null names from submitter API

This PR updates the submitters query to not include null results for submitters

Test Steps:
1. Confirm that this still returns results
```sh
curl --location 'http://localhost:7071/api/v1/receivers/ignore.HL7/deliveries/submitters/search' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer dummy' \
--data '{
    "sort": {
        "direction": "ASC",
        "property": "type"
    },
    "pagination": {
        "page": 1,
        "limit": 100
    },
    "filters": [

    ]
}'
```

## Changes
- Filters out submitters with null names
- Fixes how the location is generated for the submitter type

## Checklist

### Testing

### Process

## Linked Issues


## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
